command/build.go

```diff
@@ -392,29 +414,41 @@ func buildRun(cmd *cobra.Command, args []string) {
 	var uploader scheduler.Manager
 
 	repo := MustFindRepository()
-	startTs := time.Now()
-
-	apps = mustArgToApps(repo, args)
 
 	if !buildSkipUpload || !buildForce {
 		store = MustGetPostgresClt(repo)
 	}
 
-	if !buildForce {
-		fmt.Printf("Evaluting build status of applications:\n")
+	startTs := time.Now()
+
+	apps = mustArgToApps(repo, args)
+	baur.SortAppsByName(apps)
+
+	fmt.Printf("Evaluating build status of applications:\n")
+	if buildForce {
+		apps = appsWithBuildCommand(apps)
+	} else {
 		apps = pendingBuilds(store, apps)
 	}
 
+	fmt.Println()
+	fmt.Printf("Building applications with build status: %s\n",
+		coloredBuildStatus(baur.BuildStatusPending))
+
+	if buildSkipUpload {
+		fmt.Println("Outputs are not uploaded.")
+	}
+
 	if len(apps) == 0 {
-		fmt.Println()
 		term.PrintSep()
-		fmt.Println("Application build(s) already exist, nothing to build.\n" +
-			"If you want to rebuild applications pass '-f' to 'baur build'")
+
+		if !buildForce && len(apps) > 0 {
```

Comment:
`len(apps)` is always `0` because of the outer `if` statement

Reply:
thanks, fixed